test/request-response-samples/execFile__valid_xml_generated_on_request_with_complex_type_element_form_default_missing/request.json

```diff
@@ -1,8 +1,6 @@
 {
-  "params" : {
     "fileName": "PSTN",
```

Comment:
Are there tests that cover RPC style web services?  I feel the approach we've been taking with additional features / fixes is to add new tests with the goal of leaving old ones in tact.  That way we ensure no breaking changes for existing users.